utils/Room.js

```diff
@@ -0,0 +1,31 @@
+module.exports = class Room{
+    #private = 123
+    constructor(){
+        this.listOfPlayers = []
+        this.nextPlayer = 0;
+    }
+    incrementPlayer(){
+        this.nextPlayer++;
+        if (this.listOfPlayers.length == this.nextPlayer){
+            this.nextPlayer = 0
+        }
+    }
+    nextTurn(){
```

Comment:
next turn does not sound like it returns anything, so it should return null

you should make a separate function that gets Player

so you do:
```
game.nextTurn();
nextPlayerNumber = game.getNextPlayerNumber()
```

notice how the function names describes exactly what is going on